packages/core/src/Input/Input.spec.js

```diff
@@ -23,4 +24,42 @@ describe('Input', () => {
     ).toJSON()
     expect(json).toMatchSnapshot()
   })
+  test('it renders an input element with helper text', () => {
+    const helperText = 'hello world'
+    const paletteTheme = createTheme(theme)
+    const { getByText, rerender } = render(
+      <Input
+        id={id}
+        helperText={<Input.ErrorText>{helperText}</Input.ErrorText>}
+      />
+    )
+    let helperTextNode = getByText(helperText)
+    expect(helperTextNode).not.toBeNull()
+    expect(helperTextNode).toHaveStyleRule(
+      'color',
+      getPaletteColor('error.base')({ theme: paletteTheme })
```

Comment:
This is a much better way to test that colours match, I wonder if we can make a testing specific helper for this.